package.json

```diff
@@ -16,6 +17,15 @@
     "morgan": "^1.10.0"
   },
   "devDependencies": {
+    "chai": "^4.3.10",
+    "chai-http": "^4.4.0",
+    "eslint": "^8.52.0",
+    "eslint-config-semistandard": "^17.0.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.0",
+    "eslint-plugin-n": "^15.7.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "mocha": "^10.2.0",
```

Comment:
Thank you for adding these! There's one more step to get ESlint reporting on the updated rules, which is to `extend` these configs in the `.eslintrc`

I opened a PR into your branch with these changes: 

https://github.com/Code-the-Dream-School/ee-prac-team2-back/pull/2